pkg/client/client.go

```diff
@@ -65,6 +67,11 @@ type Interface interface {
 	ClusterRoleBindingsInterface
 	ClusterResourceQuotasInterface
 	AppliedClusterResourceQuotasNamespacer
+	// Openshift specific methods.
+	Discovery() discovery.DiscoveryInterface
+	SetTimeout(d time.Duration)
+	// Implements kubectl RESTClient
+	resource.RESTClient
```

Comment:
It would be better to add a method that returns the RESTClient interface like upstream
https://github.com/kubernetes/kubernetes/blob/master/pkg/client/clientset_generated/clientset/typed/core/v1/core_client.go#L28

/cc @deads2k